submissions/MosQuitO404/js-dom/style.css

```diff
@@ -0,0 +1,527 @@
+@import url('https://fonts.googleapis.com/css2?family=Titillium+Web:wght@400;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+html {
+    background-image: url(./teams_img/ferrari-sf15-t-formula-1-f1.jpg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-attachment: fixed;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    min-width: 375px;
+    max-width: 1540px;
+}
+
+@media screen and (max-width: 760px) {
+    html {
+        background-image: none;
+        background-color: #15151E;
```

Comment:
Try to use CSS3 Vars for keeping your colours in one place. This is good practice handling colours/typography in one place.

https://www.w3schools.com/css/css3_variables.asp